package.json

```diff
@@ -92,6 +93,7 @@
     "@types/tmp": "0.1.0",
     "@types/uuid": "^3.4.4",
     "@types/xdg-basedir": "^2.0.0",
+    "@types/yargs": "^13.0.1",
```

Comment:
Trying hard to keep our dependencies down.  Do we strictly need yargs here for something simple only we're ever going to run?  @bcoe your call :)

Reply:
As long as it's a development dependency, I like that it creates a self documenting CLI; I think anything for the benchmarker should be installed with `--save-dev`.